api/Cache.json

```
@@ -17,7 +17,7 @@
           },
           "firefox": {
             "version_added": "39",
-            "notes": "Service workers (and <a href='https://developer.mozilla.org/docs/Web/API/Push_API'>Push</a>) have been disabled in the <a href='https://www.mozilla.org/firefox/organizations/'>Firefox 45 & 52 Extended Support Releases</a> (ESR)."
+            "notes": "<a href='https://www.mozilla.org/en-US/firefox/organizations/'>Extended Support Releases (ESR)</a>: Firefox 78 is the first ESR release that supports Service workers (and the Push API). Earlier ESR releases had no support."
```

I thought this was a little bit indirect. What do you think of something like this?

```suggestion
            "notes": "<a href='https://www.mozilla.org/en-US/firefox/organizations/'>Extended Support Releases (ESR)</a> before Firefox 78 ESR do not support service workers and the Push API."
```

---

Thanks, that sounds better. Updated!